src/function/list/list_slice_function.cpp

```diff
@@ -9,23 +9,22 @@ namespace kuzu {
 namespace function {
 
 static void normalizeIndices(int64_t& startIdx, int64_t& endIdx, uint64_t size) {
+    int64_t sSize = size;
```

Comment:
Do we actually need this because size is an invariant in this function as I understand.

Reply:
Removed with https://github.com/kuzudb/kuzu/pull/5573/commits/879b67ab0554dabb39b67f1b5a7fd8c64455eae1